phpBB/posting.php

```diff
@@ -1238,6 +1238,7 @@
 	*
 	* @event core.posting_modify_submission_errors
 	* @var	array	post_data	Array with post data
+	* @var	array	poll		Array with poll data from post (must be used instead of the post_data equivalent)
```

Comment:
please add `@changed` to the event, see https://github.com/phpbb/phpbb/blob/740be0507e209afda5eb954ab674e557c2d8f8f6/phpBB/includes/ucp/ucp_prefs.php#L72